src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -7505,4 +7505,46 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_BadCaseInSwitchArm" xml:space="preserve">
     <value>A switch expression arm does not begin with a 'case' keyword.</value>
   </data>
+  <data name="WRN_CallNotInterceptable" xml:space="preserve">
+    <value>Cannot intercept a call to '{0}' because it is not marked with 'System.Runtime.CompilerServices.InterceptableAttribute'.</value>
```

Comment:
>Cannot [](http://example.com/codeflow?start=11&length=6)

nit: "Cannot" seems too strong here, given this is a warning. Consider "Call to ... is being intercepted but the method is not marked as interceptable" or some such.